.travis.yml

```diff
@@ -47,4 +46,4 @@ script:
   - python -V
   - which python3
   - python3 -V
-  - travis-wait-improved --timeout=180m scripts/travis/ci.sh
```

Comment:
What was the reason for removing this?

Reply:
Not sure, but @nicknezis fixed an issue with the TravisCI 🥊 Python and that might line may have been removed during the debug process.

Reply:
Yes, that script was causing the issue due to Python dependencies. I tested without the script and it worked. I think the default behavior is to terminate a build if there is no output for more than 10 minutes. I'm not sure if we have this issue in our build. If we do need to put it back, then we will have to resolve the Python issues.